documentation/contributor_doc.md

```diff
@@ -0,0 +1,57 @@
+# Nexus Replication Plugin
+
+[https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin)
+
+This is Nexus plugin that provides replication logic when new (Maven) artifacts are stored in a hosted repository. 
```

Comment:
This plugin does _not_ provide replication logic. Replication is provided by Nexus core. The plugin just watch for uploads and notifies peers, reveives notification and triggers download to proxy repositories.